app/blog/2017-07-12-an-update-on-neon-animation.md

```diff
@@ -0,0 +1,70 @@
+
+Sharing is easy if you restrict your animation parameters to something like keyframes. If the goal is instead to share an animation concept with infinite flexibility, then you’d essentially be exposing the entire animations API and reinventing the wheel.
+
+## Alternatives
+As I've mentioned, animations have not gone away, they're still here in the platform. There are 3 different alternatives that we suggest (in no particular order):
+
+1. CSS Keyframe Animations
+2. Web Animations API
+3. CSS Transitions
+
+### CSS Keyframe Animations
+CSS Keyframe animations are great because they provide great control over animations. Ways they do this: they introduce the concept of a Keyframe which essentially represent a CSS snapshot at a specific percentage of an animation's timeline, CSS Keyframes also provide a nice, declarative syntax to define an animation by stylistically decoupling animations into the CSS. Another nice thing about CSS Keyframes is that they have been [supported](http://caniuse.com/#search=css%20animation) (prefixed) since Obama's first term (unprefixed around when [The Weeknd](https://www.youtube.com/watch?v=KEI4qSrkPAs) stopped feeling their face).
+
+What makes them not so great is trying to make changes to the keyframes; it's possible but it's ugly, painful, and not always efficient. This makes it difficult to make “dynamic” animations that calculate keyframes.
+
+The basics of CSS Keyframe animations are well-covered by [Mozilla](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations/Using_CSS_animations).
+
+### Web Animations API
+The [Web Animations API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API) is a great all-around replacement as `neon-animation` was designed around the Web Animations API. It  does everything the CSS Keyframe Animations do, but it also has the luxury of being in Javascript and thus can make changes on the fly. The web animations API is natively supported in Chrome & Firefox, with a [polyfill](https://github.com/web-animations/web-animations-js) required in other browsers [other browsers](http://caniuse.com/#search=web%20animations%20api).
+
+[Mozilla](https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API/Using_the_Web_Animations_API) has a good intro to the API.
+
+### CSS Transitions
+CSS Transitions allow you to animate changes in CSS properties. Transitions can also easily be set via javascript so you don't run into the same issues as CSS Keyframe animations.
+
+The largest limitation of CSS Transitions is that they make it difficult to coordinate animations involving multiple properties or multiple states.
+
+Here's a trusty [MDN link](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Transitions/Using_CSS_transitions) on how to use them.
```

Comment:
Yep, a separate post (or posts) is a good idea. It would be nice to let folks know it's coming. 

The sum-up at the end is just a nice way of reminding people what the take-aways from your article are. You could also use it to tease the future article, so people are left anticipating it, instead of going "now what do I do?"

## Conclusion

To sum up:

-   If you're using another element from the PolymerElements org that depends on `neon-animation`, like `iron-dropdown` or `paper-menu-button`, you don't have to do anything. We'll replace these animations.

-   If you're creating a new element from scratch, we recommend using CSS keyframe animations, CSS transitions, or the Web Animations API.

-   If you're using `neon-animation` directly, you should start planning to eventually move off it, and use one of the platform's animation APIs instead.

Still not sure what these APIs look like? We'll share some examples in a future blog post.